src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -293,9 +293,13 @@ object DemuxFastqs {
       |
       |`--fastq-skip-read-numbers=true --fastq-include-sample-barcodes=true --illumina-file-names=true`
       |
-      |To output reads with a passing filter flag, use:
+      |To output reads with a passing filter flag, as encoded in the <is filtered> field in the header comment, use:
```

Comment:
Adding backticks, since the usage is parsed as markdown, which is really nice.
```suggestion
      |To output reads with a passing filter flag, as encoded in the `<is filtered>` field in the header comment, use:
```

Reply:
Also, I think you're mixing up PRs, as I think I asked for this in the other PR: https://github.com/fulcrumgenomics/fgbio/pull/713#discussion_r712400396